service/application/src/test/java/com/solace/maas/ep/event/management/agent/commandManager/SempDeleteCommandManagerTest.java

```diff
@@ -688,6 +689,102 @@ void testNotFoundDeleteRdpRestConsumer() throws ApiException {
         assertThat(cmd.getResult().getStatus()).isEqualTo(JobStatus.success);
     }
 
+    // test DeleteRdpOauthJwtClaim
```

Comment:
Can you put these tests under a `@Nested` class? That way related tests are grouped and the test names can be much smaller! e.g. `testDeleteRdpOauthJwtClaimHappyPath` -> `testHappyPath`

Reply:
Good suggestion. Adding 'test' to every test is redundant as well.